### DISCUSSION TITLE
Suggestion: Be able to hide streaming icons+Sorting

### DISCUSSION THREAD

**Discussion by Tsunamijhoe** [trakt/showly#503] [Ideas / Feature Requests]
I would really like to see an option to remove/hide the streaming icons added in latest update. They are pervasive and annoying.
I would also like a way for the progress sorting to stop moving shows that i have caught up on to the bottom,so i just get a simple alphabetical list of all active shows i can scroll.

**Comment by blackholeearth** [comment]
> I would really like to see an option to remove/hide the streaming icons added in latest update. They are pervasive and annoying. 

No1:  is done  -   

![ZomboDroid_hide streaming ](https://github.com/michaldrabik/showly-2.0/assets/7512105/98d863ce-67ef-4a2d-9b6d-78fd61f23557)

-----




> I would also like a way for the progress sorting to stop moving shows that i have caught up on to the bottom,so i just get a simple alphabetical list of all active shows i can scroll.

 No2: There is now **onHold section**. İf you havent watched a show like 2years. 

U can **long press show > pin to onhold**


------

İ think u can close this issue.
 (Which is actually 2 issues in 1)

**Reply by Tsunamijhoe** [reply]
Not agreeing that issue 2 is done.. The "on hold" function is too cumbersome and not really applicable to what i would like. 
The progress and watchlist should either be merged, or let you move shows from any list to your progress list, and the progress list should not auto move shows to other tabs when you have watched all episodes, it should be a user choice to auto move or manual move.  That would be perfect for me.
For example, i watch X-Files regularly, and i have to go to collection -> Watchlist to record progress which is annoying.
What i need is a simple progress list with ALL the shows i watch, regardless if they are current or old, and it needs to be sorted alphabetically as minimum, and i need to be able to add or delete shows as i see fit to said progress list.

Thank you for ability to hide streaming icons.
